library/common/jni/jni_utility.cc

```diff
@@ -299,6 +302,8 @@ jbyteArray ToJavaByteArray(JNIEnv* env, const std::string& str) {
 
 void JavaArrayOfByteArrayToStringVector(JNIEnv* env, jobjectArray array,
                                         std::vector<std::string>* out) {
+  ASSERT(out);
+  ASSERT(array);
```

Comment:
not sure whether this was added as part of this PR intentionally

Reply:
They are intentional. I just found neither the caller nor callee check these.